beyond-essence/inventory/error-messages.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: beyond-essence
+header_image: private/main.jpg
+title: Error messages
+background_image: index.jpg
+---
+## General copywriting guidelines
+
+Error messages inform the merchant that something went wrong and let them know what they can do about it. The wording should be simple and straightforward and use terms that the merchant can easily understand. Try to avoid technical terms and words that may have a negative connotation, e.g. "invalid", "error", "wrong", "failed", "system", "validation", etc. 
+
+Keep in mind that each error message interrupts the merchant's workflow and might lead to the merchant getting frustrated and giving up on their current action, especially when the message is unfriendly and/or unclear. So when writing the text for an error message, focus on the issue at hand and give the merchant helpful guidance on what to do next.
+
+**Be specific**: Whenever possible, use a specific error message for each use case to be as helpful for the merchant as possible. Try to avoid generic messages like "Something went wrong" (What went wrong? Why? Is there anything the merchant can do to make it right?) or "The entered value is invalid" (Why is it invalid? What are the requirements for a valid entry?). 
+If a message really needs to be generic, because it needs to be used for different use cases or because the error can't be defined any further, write it at least as friendly and helpful as possible.
+
+**Be concise**: Error messages appear in boxes with restricted space. Also, when the merchant receives an error message, they are in the middle of performing an action that is being interrupted by the error. They want to resume their work as quickly as possible and don't want to read lengthy explanations, apologies or instructions.
+Brevity should not keep the message from sounding friendly and helpful, though. Be concise, but don't sound like a robot.
+
+**Be friendly & positive**: Instead of telling the merchant what they can *not* do, be positive and tell them what they *can* do.
+Even when the message needs to be short, don't make it sound threatening or commanding and don't make the merchant feel bad about having made a mistake.
+
+DO: You can add max. %{count} products.
+
+DON'T: You can't add more than %{count} products.
+
+DO: Incorrect password. Maybe you mistyped?
```

Comment:
Is this an existing message?

Reply:
No, but this would be the kind of error message I would like to see more, but this is something we can discuss when the error messages are reworked.